extras/data/CHORUSPb/metadata.yaml

```diff
@@ -2,9 +2,19 @@ arxiv: null
 y_label: sigr_ch_pb
 y_label_tex: $\sigma^\mathrm{CHORUS}_{\mathrm{red},\mathrm{Pb}}$
 hepdata: 10.17182/hepdata.6187.v1
+
 CHORUS_CC_NB_PB_SIGMARED:
   description: CHORUS reduced cross section antineutrino-nucleon scattering on lead
   nnpdf_id: CHORUSNBPb
+
 CHORUS_CC_NU_PB_SIGMARED:
   description: CHORUS reduced cross section neutrino-nucleon scattering on lead
   nnpdf_id: CHORUSNUPb
+
+CHORUS_NU_PB:
```

Comment:
why you need a copy?

Reply:
(similar for others)

Reply:
It is to make sure that the grids are generated assuming $ep$. If we'd fit the bound nucleus PDF instead (which is what we should do in the future) then they will be same.

Reply:
should it then not be called `EL_P` or similar?